### DISCUSSION TITLE
Download URL not working

### DISCUSSION THREAD

**Discussion by ExcChua** [community/community#21789] [New to GitHub]
<p>Hi,</p>
<p>I’m trying to download a release that I created, in order to test the URL.</p>
<p>The URL is: <a href="https://github.com/Eximchain/BackupLambda/releases/download/0.1/BackupLambda" rel="nofollow noopener">https://github.com/Eximchain/BackupLambda/releases/download/0.1/BackupLambda</a></p>
<p>I can get it to work from Firefox, but if I use wget <a href="https://github.com/Eximchain/BackupLambda/releases/download/0.1/BackupLambda" rel="nofollow noopener">https://github.com/Eximchain/BackupLambda/releases/download/0.1/BackupLambda</a> I get a 404.</p>
<p>What’s happening, and how do I get it to work?</p>

**Comment by CleanCut** [comment, marked as answer]
<p>That repository does not appear to be configured as public. You are able to download it with your browser because you have logged into the site with your browser.  If you are okay with setting your repository to be public (anyone would be able to see it), then your unauthenticated wget client would be able to download the release.</p>

**Comment by suryadey-1202** [comment]
<p>My repository is public. It still doesn’t work.</p>
<p>I can clone it from any place. But wget is not working. </p>
<p>curl -LJO <a href="https://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">https://github.com/suryadey-1202/test_data/mock_demo.json</a> (working)</p>
<p>With HTTPS:</p>
<p>wget --no-check-certificate --content-disposition <a href="https://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">https://github.com/suryadey-1202/test_data/mock_demo.json</a><br>
–2020-03-25 17:09:03-- <a href="https://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">https://github.com/suryadey-1202/test_data/mock_demo.json</a><br>
Resolving <a href="http://github.com" rel="nofollow noopener">github.com</a> (<a href="http://github.com" rel="nofollow noopener">github.com</a>)… 140.82.112.4<br>
Connecting to <a href="http://github.com" rel="nofollow noopener">github.com</a> (<a href="http://github.com" rel="nofollow noopener">github.com</a>)|140.82.112.4|:443… connected.<br>
HTTP request sent, awaiting response… 404 Not Found<br>
2020-03-25 17:09:04 ERROR 404: Not Found.</p>
<p>wget <a href="https://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">https://github.com/suryadey-1202/test_data/mock_demo.json</a><br>
–2020-03-25 17:22:45-- <a href="https://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">https://github.com/suryadey-1202/test_data/mock_demo.json</a><br>
Resolving <a href="http://github.com" rel="nofollow noopener">github.com</a> (<a href="http://github.com" rel="nofollow noopener">github.com</a>)… 140.82.113.4<br>
Connecting to <a href="http://github.com" rel="nofollow noopener">github.com</a> (<a href="http://github.com" rel="nofollow noopener">github.com</a>)|140.82.113.4|:443… connected.<br>
HTTP request sent, awaiting response… 404 Not Found<br>
2020-03-25 17:22:45 ERROR 404: Not Found.</p>
<p>With HTTP:</p>
<p>wget --no-check-certificate --content-disposition <a href="http://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">http://github.com/suryadey-1202/test_data/mock_demo.json</a><br>
–2020-03-25 17:16:02-- <a href="http://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">http://github.com/suryadey-1202/test_data/mock_demo.json</a><br>
Resolving <a href="http://github.com" rel="nofollow noopener">github.com</a> (<a href="http://github.com" rel="nofollow noopener">github.com</a>)… 140.82.113.4<br>
Connecting to <a href="http://github.com" rel="nofollow noopener">github.com</a> (<a href="http://github.com" rel="nofollow noopener">github.com</a>)|140.82.113.4|:80… connected.<br>
HTTP request sent, awaiting response… 301 Moved Permanently<br>
Location: <a href="https://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">https://github.com/suryadey-1202/test_data/mock_demo.json</a> [following]<br>
–2020-03-25 17:16:02-- <a href="https://github.com/suryadey-1202/test_data/mock_demo.json" rel="nofollow noopener">https://github.com/suryadey-1202/test_data/mock_demo.json</a><br>
Connecting to <a href="http://github.com" rel="nofollow noopener">github.com</a> (<a href="http://github.com" rel="nofollow noopener">github.com</a>)|140.82.113.4|:443… connected.<br>
HTTP request sent, awaiting response… 404 Not Found<br>
2020-03-25 17:16:02 ERROR 404: Not Found.</p>

**Reply by 124dfg** [reply]
تمام او تاكد من الاتصال بالإنترنت غالبا

**Comment by mariamartens21** [comment]
I have facing the same issues on my client site plz [visit](https://www.snapofficial.net/).
